src/javascripts/app.js

```diff
@@ -0,0 +1,10 @@
+import LoginController from "./controllers/loginController";
+import AccountService from "./services/accountService";
+import LoginView from "./views/loginView";
+
+    document.addEventListener("DOMContentLoaded", () => {
```

Comment:
@HoangLiarPhuongLe Why does this code line indent with tab but not at the beginning of the line?